tutor4points/core/templates/tutors.html

```diff
@@ -42,9 +44,43 @@
 
 
     <div>
-        <button type ="submit" name="request" value="request" class="request-button">Request</button>
+        <button type ="submit" name="request" value="request" class="request-button" data-toggle= "modal" data-target="#tutor-request-modal">Request</button>
+    </div>
+
+    <!-- insert request form here -->
+    <div class = "modal fade" id = "tutor-request-modal">
```

Comment:
It will probably be better if you move the modal to outside all the divs, so the styling won't get affected by the outside styles in the future if more are added.